phpBB/config/default/container/services_files.yml

```diff
@@ -1,4 +1,35 @@
 services:
+    attachment.delete:
```

Comment:
actually we have 3 things here:
- filesystem
- upload
- attachments

I think we should split them